main.go

```diff
@@ -159,6 +161,24 @@ func main() {
 		if feature.Gates.Enabled(feature.EKS) {
 			setupLog.Info("enabling EKS controllers")
 
+			enableIAM := feature.Gates.Enabled(feature.EKSEnableIAM)
+			allowAddRoles := feature.Gates.Enabled(feature.EKSAllowAddRoles)
```

Comment:
I don't think this is used in the manager.....i'll create a new issue to remove it

Reply:
Issue created #2001